From 5261d17a8c315ad306e734ea00cad3c7dc7f50ba Mon Sep 17 00:00:00 2001
From: James Bonfield <jkb@sanger.ac.uk>
Date: Tue, 16 Mar 2021 18:30:53 +0000
Subject: [PATCH] Fix cram_bambam.c once more.

Fix silly mistake in 4df5bbd.  io_lib/os.h was included so it could
define CRAM_IO_CUSTOM_BUFFERING, so obviously it has to be before the
ifdef.
---
 io_lib/cram_bambam.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/io_lib/cram_bambam.c b/io_lib/cram_bambam.c
index 2e85fbf..4a16263 100644
--- a/io_lib/cram_bambam.c
+++ b/io_lib/cram_bambam.c
@@ -60,11 +60,12 @@
 #include "io_lib_config.h"
 #endif
 
+#include "io_lib/os.h"
+
 #if !defined(CRAM_IO_CUSTOM_BUFFERING)
 // not used
 #else
 
-#include "io_lib/os.h"
 #include "io_lib/cram_bambam.h"
 
 //-----------------------------------------------------------------------------
-- 
2.31.0

